api/src/main/java/org/eclipse/microprofile/metrics/Snapshot.java

```diff
@@ -64,7 +64,10 @@ public abstract class Snapshot {
      *
      * @return an array of {@link HistogramBucket} if it is available or an empty array if not available
      */
-    public abstract HistogramBucket[] bucketValues();
+    @aQute.bnd.annotation.baseline.BaselineIgnore("5.2")
```

Comment:
Why is the baseline package version "5.2"? 

I know that is the version of the _plug-in_ being used, but the Javadoc for the `value` attribute for the annotation states
> Baseline package version.
> The version must be a valid OSGi version string.

That doesn't clarify much, but It seems that this value is intended to reflect the version of the component being produced, not the version of the `bnd baseline` plug-in being used.